requirements.yaml

```diff
@@ -0,0 +1,8 @@
+dependencies:
+  - alias: operator
+    name: entando-k8s-controller-coordinator
+    repository: file://../entando-k8s-controller-coordinator/charts/entando-k8s-controller-coordinator
```

Comment:
Looks hardcoded. Is this correct?

Reply:
Unfortunately I had to make changes locally to the entando-k8s-controller-coordinator Helm template. They will be merged into master once we tackle ENG-2227 which would parameterize image version. Once this is in place, we control the final release almost completely from entando-k8s-operator-bundle. I will then also fix this line. 

Note that we will probably not be running these helm templates from a pipeline at any stage as it is quite an important step of our release process.